frappe/public/js/frappe/widgets/links_widget.js

```diff
@@ -24,6 +25,22 @@ export default class LinksWidget extends Widget {
 			this.options.links = this.links;
 		}
 		this.widget.addClass("links-widget-box");
+
+		if (this.description) {
+			const description = $(`
+				<button class="btn-reset card-description-btn ml-2">
```

Comment:
Added `ml-2` because the inner workspace links also have the same margin. Also, the tooltip could be black for a better contrast but we have used light tooltips everywhere:

<img width="340" alt="image" src="https://github.com/frappe/frappe/assets/24353136/c522f78f-de82-4631-8916-d05cefce7ba3">